Signal/src/call/CallAudioService.swift

```diff
@@ -32,7 +32,7 @@ import Foundation
 
     internal func stateDidChange(call: SignalCall, state: CallState) {
         AssertIsOnMainThread()
-        self.handleState(state)
+        self.handleState(call:call)
```

Comment:
Why this change?

Previously we were passing an immutable state to this method. But now we're passing something for which the state could change (on a different thread).

(also nit: space after `:` per linter)